tests/test_notes.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015-2019 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>..
+#
+# Authors:
+#     Santiago Dueñas <sduenas@bitergia.com>
+#
+
+import os
+import unittest
+import unittest.mock
+
+import click.testing
+
+from release_tools.notes import notes
+from release_tools.entry import CategoryChange
+
+
+RELEASE_NOTES_CONTENT = """## release-tools 0.8.10 - (2019-01-01)
+
+**New features:**
+
+ * first feature (#1)\\
+   Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+   eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+   minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+   aliquip ex ea commodo consequat. Duis aute irure dolor in
+   reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+   pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+   culpa qui officia deserunt mollit anim id est laborum.
+ * second feature (#3)
+ * final feature
+
+**Bug fixes:**
+
+ * first bug fix (#2)
+ * another fix\\
+   Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+   eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+   minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+   aliquip ex ea commodo consequat. Duis aute irure dolor in
+   reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+   pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+   culpa qui officia deserunt mollit anim id est laborum.
+"""
+RELEASE_NOTES_EMPTY = """## release-tools 0.8.10 - (2019-01-01)
+
+No changes list available.
+"""
+RELEASE_NOTES_FILE_ALREADY_EXISTS_ERROR = (
+    "Error: Release notes for version 0.8.10 already exist. Use '--overwrite' to replace it."
+)
+CHANGELOG_DIR_NOT_FOUND = (
+    r"Error: changelog entries directory .+ does not exist"
+)
+INVALID_NAME_ERROR = (
+    "Error: Invalid value for \"NAME\": cannot be empty"
+)
+INVALID_VERSION_ERROR = (
+    "Error: Invalid value for \"VERSION\": cannot be empty"
+)
+
+
+class TestNotes(unittest.TestCase):
+    """Unit tests for notes script"""
```

Comment:
A test is missing to check the behavior when something goes wrong when reading the changelog entries (https://github.com/Bitergia/release-tools/pull/3/files#diff-1502d288e8965b06c9dae8dd051b2710R103)

Reply:
Fixed.